dependencies.txt

```diff
@@ -211,4 +211,5 @@ github4s: base64, better-monadic-for, blaze, boopickle, cats, cats-effect, cats-
 scala-pet-store: better-monadic-for, blaze, boopickle, cats, cats-effect, cats-effect-testing, circe, circe-config, circe-generic-extras, claimant, cloc-plugin, discipline, discipline-scalatest, discipline-specs2, doobie, enumeratum, fansi, fastparse, fastparse-scalameta, fs2, http4s, http4s-parboiled2, jawn, jawn-fs2, json4s, kind-projector, log4cats, log4s, mdoc, metaconfig, minitest, munit, paiges, play-json, portable-scala-reflect, pprint, pureconfig, refined, scala, scala-collection-compat, scala-js-stubs, scala-parser-combinators, scala-xml, scalacheck, scalameta, scalapb, scalatags, scalatest, scalatest-3-0, scalatestplus-scalacheck, scodec-bits, sconfig, shapeless, silencer, simulacrum, sourcecode, specs2, specs2-more, spray-json, tsec, tut, twirl, unique, utest, vault, wartremover
 pureconfig-more: akka, better-monadic-for, blaze, boopickle, cats, cats-effect, cats-effect-testing, circe, claimant, cloc-plugin, discipline, discipline-scalatest, discipline-specs2, enumeratum, fansi, fastparse, fastparse-scalameta, fs2, genjavadoc, http4s, http4s-parboiled2, jawn, jawn-fs2, json4s, kind-projector, log4cats, log4s, magnolia, mdoc, mercator, metaconfig, minitest, munit, paiges, play-json, portable-scala-reflect, pprint, pureconfig, refined, scala, scala-collection-compat, scala-java8-compat, scala-js-stubs, scala-parser-combinators, scala-xml, scalacheck, scalacheck-shapeless, scalameta, scalapb, scalatags, scalatest, scalatest-3-0, scalatestplus-junit, scalatestplus-scalacheck, scodec-bits, sconfig, shapeless, silencer, simulacrum, sourcecode, specs2, specs2-more, spray-json, tut, twirl, unique, utest, vault, wartremover
+coulomb: akka, algebra, better-monadic-for, blaze, boopickle, cats, cats-effect, cats-effect-testing, cats-testkit-scalatest, circe, claimant, cloc-plugin, discipline, discipline-scalatest, discipline-specs2, enumeratum, fansi, fastparse, fastparse-scalameta, fs2, http4s, http4s-parboiled2, jawn, jawn-fs2, json4s, kind-projector, log4cats, log4s, machinist, magnolia, mdoc, mercator, metaconfig, minitest, munit, paiges, play-json, portable-scala-reflect, pprint, pureconfig, pureconfig-more, refined, scala, scala-collection-compat, scala-java8-compat, scala-js-stubs, scala-parser-combinators, scala-xml, scalacheck, scalacheck-shapeless, scalameta, scalapb, scalatags, scalatest, scalatest-3-0, scalatestplus-junit, scalatestplus-scalacheck, scodec-bits, sconfig, shapeless, silencer, simulacrum, singleton-ops, sourcecode, specs2, specs2-more, spire, spray-json, tut, twirl, unique, utest, vault, wartremover
```

Comment:
@erikerlandson I suspect that after https://github.com/erikerlandson/coulomb/pull/75, this list wouldn't be so long